### ISSUE TITLE
Question about the generator GMMN

### ISSUE THREAD

**Issue by Daisy-GENG** [valeoai/3DGenZ#3]
Thank you for publishing your code of such an interesting work. I am new to ZSL/G-ZSL, so I am little bit confused about the source of word representations(Word2Vec/GloVe) of the SemanticKITTI dataset that you provided in this repo. Do you manually build up the corpus of SemanticKITTI dataset, and use Word2Vec/GloVe to get these word representations, or is it publicly available? 

Thank you so much!!

Best regards,
RUOYU GENG

**Comment by BjoernMichele** [comment]
Thank you very much for your interest in our work. 

The word representations for the semantic segmentation datasets are obtained from publicly available word2vec and GloVe models. So we querried the pre-trained word2vec/GloVe models with the class names to obtain the vector representations for our classes. I have to double check but for W2V I am pretty sure we used the model from the gensim library (https://radimrehurek.com/gensim/models/word2vec.html), and for GloVe the models provided  on the official website (https://nlp.stanford.edu/projects/glove/).

Best regards, 
Bjoern
